src/frontend/src/pages/new-challenge/new-challenge.tsx

```diff
@@ -1,7 +1,121 @@
+                    container
+                    direction="row"
+                    justify="space-evenly"
+                    alignItems="stretch"
+                    spacing={3}
+                >
+                    <Grid item xs={4}>
+                        <Typography>Actual Definition</Typography>
+                    </Grid>
+                    <Grid item xs={4}>
+                        <Card>
+                            <CardContent>
+                                <Typography gutterBottom>
+                                    Optional Definition1
+                                </Typography>
+                            </CardContent>
+                            <CardActions>
+                                {/* Create an 'onClick' method that calls the API */}
+                                <Button size="small" color="primary">
+                                    Change
+                                </Button>
+                            </CardActions>
+                        </Card>
+                    </Grid>
+                    <Grid item xs={4}>
+                        <Card>
+                            <CardContent>
+                                <Typography gutterBottom>
+                                    Optional Definition2
+                                </Typography>
+                            </CardContent>
+                            <CardActions>
+                                {/* Create an 'onClick' method that calls the API */}
+                                <Button size="small" color="primary">
+                                    Change
+                                </Button>
+                            </CardActions>
+                        </Card>
+                    </Grid>
+                </Grid>
+                {/*  spacing */}
+                <div style={{ padding: 50 }}></div>
+                <Typography gutterBottom variant="h4">
+                    3 - Input your sentence
+                </Typography>
+                <div style={{ padding: 50 }}>
+                    <TextField
+                        fullWidth
+                        multiline
+                        rows="4"
+                        variant="outlined"
+                    />
+                </div>
+                <div style={{ padding: 50 }}></div>
+                <Typography gutterBottom variant="h4">
+                    4 - Send to some friends
+                </Typography>
+                <Autocomplete
+                    multiple
+                    options={friends}
+                    getOptionLabel={friends => friends}
+                    renderInput={params => (
+                        <TextField
+                            {...params}
+                            variant="standard"
+                            label="friends"
+                            placeholder="friend"
+                        />
+                    )}
+                ></Autocomplete>
```

Comment:
```suggestion
                <Autocomplete
                    multiple
                    options={friends}
                    getOptionLabel={friends => friends}
                    renderInput={params => (
                        <TextField
                            {...params}
                            variant="standard"
                            label="friends"
                            placeholder="friend"
                        />
                    )}
                />
```
You can have a single tag here since there's nothing inside.

Reply:
Would it not be more style wise if we have the term at the end?
As the similar style is also implemented for other elements.

Reply:
As far as I can see from the documentation, the Autocomplete component doesn't take any children. Thus, ```<Autocomplete />``` is more appropriate than ```<Autocomplete></Autocomplete>``` since you would only need the latter if you were to add some JSX children.

Reply:
This issue has been addressed. Please review and resolve the conversation.
Thanks for the info.